products/perl.md

```diff
@@ -12,6 +12,14 @@ activeSupportColumn: true
 releaseDateColumn: true
 eolColumn: Critical security patches
 
+identifiers:
+-   purl:  pkg:apk/alpine/perl
+-   purl:  pkg:deb/debian/perl
+-   purl:  pkg:deb/perl
+-   purl:  pkg:ipk/perl
+-   purl:  pkg:rpm/perl
```

Comment:
```suggestion
-   purl:  pkg:deb/perl
-   purl:  pkg:ipk/perl
-   purl:  pkg:rpm/perl
```

Not a big fan of those OS-less PURL, as I don't know how to decide whether they are valid or not. Should we really keep them ?